impact.html

```diff
@@ -45,60 +45,89 @@ <h5><span class="w3-opacity">DATE</span></h5>
 
         <div class="w3-justify">
           <img src="https://germini.info/wp-content/uploads/2016/12/JavaScript-if-else.jpg" alt="Girl Hat" style="width:100%" class="w3-padding-12">
```

Comment:
Please change the image into something that suits the webpage content. You may look for images at [www.pexels.com](url)